insanely_fast_whisper_rocm/core/integrations/diarization.py

```
@@ -0,0 +1,705 @@
+    """
+    with _LOCK:
+        keys_to_remove = []
+        for key, cached_pipeline in _CACHE.items():
+            device = key[1]
+            if isinstance(device, str) and device.lower() in {"cuda", "gpu"}:
+                try:
+                    import torch
+
+                    cached_pipeline.to(torch.device("cpu"))
+                except Exception:
+                    pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)